models/CalendarEntry.php

```diff
@@ -595,9 +598,14 @@ public function setUid($uid)
         $this->uid = $uid;
     }
 
+    public function hasLocation()
```

Comment:
Please add some php doc, I know its currently missing on other parts, but we need to be more strict in the future especially for public access functions/properties.

Reply:
https://github.com/humhub/humhub-modules-calendar/commit/ab5cd1db6ae7cde0032b3686d9793f24d65dc572